app/src/features/projects/CreateProjectPage.tsx

```diff
@@ -133,6 +134,9 @@ const CreateProjectPage: React.FC = () => {
   // Tracks the active step #
   const [activeStep, setActiveStep] = useState(0);
 
+  // Tracks if sections of the form have been completed
+  const [formsComplete, setFormsComplete] = useState([false, true, false, false, false, true, true, true, true]);
```

Comment:
maybe replace complete with valid to reflect that the form, as complete can imply something else

Reply:
`formStepsValid`